src/inference/src/dev/threading/cpu_streams_executor.cpp

```diff
@@ -516,6 +531,9 @@ struct CPUStreamsExecutor::Impl {
 };
 
 int CPUStreamsExecutor::get_stream_id() {
+    if (!_impl->_streams.find_thread_id()) {
+        return 0;
```

Comment:
Only an idea: is it possible to merge this checking into local()?

Reply:
I think it can't be moved to local(), because it must return a Stream() through getting or creating Stream();